doc/manual_experimental.md

```diff
@@ -2263,4 +2263,4 @@ proc makeCppClass(): NimClass {.constructor: "NimClass() : CppClass(0, 0)".} =
 
 In the example above `CppClass` has a deleted default constructor. Notice how by using the constructor syntax, one can call the appropiate constructor. 
 
-Notice when calling constructor in a global variable, it will be called before `NimMain` meaning Nim is not fully initialized.
\ No newline at end of file
+Notice when calling constructor in a global variable, it will be called before `NimMain` meaning Nim is not fully initialized.
```

Comment:
```suggestion
Notice when calling a constructor in the section of a global variable initialization, it will be called before `NimMain` meaning Nim is not fully initialized.
```